modules/ethereumNode.js

```diff
@@ -526,19 +544,12 @@ class EthereumNode extends EventEmitter {
 
                     this.emit('data', data);
 
+                    // check for startup errors
                     if (STATES.STARTING === this.state) {
                         let dataStr = data.toString().toLowerCase();
 
-                        if ('eth' === nodeType) {
-                            // (eth) prevent started until str appears
-                            if (-1 === dataStr.indexOf('jsonrpc')) {
-                                log.trace('Running eth so wait until we see JSONRPC message');
-
-                                return;
-                            }
-                        } else if ('geth' === nodeType) {
+                        if ('geth' === nodeType) {
```

Comment:
Defensive programming. If by mistake `=` is used it won't compile. Whereas if you did `nodeType = "geth"` that still compiles and will produce an undesired outcome.